index.js

```diff
@@ -269,10 +269,33 @@ function validateEmail(input, invalidLowercase) {
   return showError(input, invalidLowercase);
 }
 
+// local storage
+function saveUserDetails() {
+  const formData = new FormData(form);
+  const fullname = formData.get('fullname');
+  const email = formData.get('email');
+  const feedback = formData.get('feedback');
+  const myFormData = { name: fullname, email, feedback };
+  localStorage.setItem('myFormData', JSON.stringify(myFormData));
+}
+
 form.addEventListener('submit', (e) => {
   e.preventDefault();
   const emailValid = validateEmail(emailInput, INPUT_LOWERCASE);
   if (emailValid) {
     form.submit();
+    saveUserDetails();
   }
 });
+
+const stored = localStorage.getItem('myFormData');
+
+function getUserDetails(localObj) {
+  const userDetails = JSON.parse(localObj);
+  console.log(userDetails);
```

Comment:
- [ ] Please, don't include any debugging code (in this case the `console.log` in the repo according to [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)